### DISCUSSION TITLE
Open document in desktop app from PnP Search Results web part

### DISCUSSION THREAD

**Discussion by kboske123** [microsoft-search/pnp-modern-search#3182] [Q&A] [answered]
Hi,
I developed a custom layout to see more metadata in the search results.
But the customer also wants to open a Word document in the desktop application, instead of in the browser.
I tried the following, but the syntax doesn't seem right.

```
{{#if @item.FileType}}
  {{#eq "docx"}}
         <a href="{{slot item @root.slots.OriginalPath}}" target="_blank" style="color:{{@root.theme.semanticColors.link}}" data-interception="off">{{slot item @root.slots.FileName}}</a>
 {{else}}
         <a href="{{slot item @root.slots.OriginalPath}}?web=1" target="_blank" style="color:{{@root.theme.semanticColors.link}}" data-interception="off">{{slot item @root.slots.FileName}}</a>
 {{/eq}}
                                        {{/if}}
```
Can someone help me?
Kind regards.

**Comment by kasperbolarsen** [comment]
Hi 😁

Please review this post: https://github.com/pnp/sp-dev-solutions/issues/79 which should solve the issue 😊

**Reply by kboske123** [reply]
Thanks Kasper for pointing me to a similar issue. But my biggest issue is my If statement. How do I get to the file type?

**Comment by wobba** [comment, marked as answer]
See https://github.com/helpers/handlebars-helpers#eq and try
```
{{#eq @item.FileType "docx"}}
...
{{/eq}}

**Reply by kboske123** [reply]
That works, thank you very much!
